Resources/Locale/en-US/headset/headset-component.ftl

```diff
@@ -15,6 +15,7 @@ chat-radio-service = Service
 chat-radio-supply = Logistics
 chat-radio-syndicate = Syndicate
 chat-radio-freelance = Freelance
+chat-radio-prison = Prison
```

Comment:
I'll need you to put this in a DeltaV locale file. There's not really any need to modify upstream localizations, since the game doesn't actually care where the localizations are in the locale folders.

Reply:
this is fixed